packages/autorest.typescript/test/rlcIntegration/generated/dpgCustomization/src/pollingHelper.ts

```diff
@@ -46,6 +46,8 @@ export async function getLongRunningPoller<TResult extends HttpResponse>(
     }
   };
 
+  // enable resolveOnUnsuccessful flag by default
```

Comment:
nitpick: is generating this comment necessary?

Reply:
will remove that